smu/parser/smu_utils_lib.py

```diff
@@ -773,20 +782,16 @@ def _conformer_source(conf):
 # conformer.
 MERGE_CONFLICT_FIELDS = [
     'conformer_id',
-    'error_nstat1_1',
-    'error_nstatc_1',
-    'error_nstatt_1',
-    'error_frequencies_1',
+    'error_nstat1',
+    'error_nstatc',
+    'error_nstatv',
+    'error_nstatt'
```

Comment:
Missing comma at end of string. This leads to an implicit concat between this and the next line. Please review and fix.

Reply:
Good catch, fixed with 899e99b